airplanedev/lib
# Pass query params to View

## Description

Get the query params and pass it to the exported View.

Resolves PDS-1470

## Test plan

* Build CLI locally using local copy of lib
* Init an inline View
* Add a console.log on the params prop
* airplane dev with built CLI
* Verify that the query params are printed

<img width="285" alt="image" src="https://user-images.githubusercontent.com/1384161/215888252-68509223-2dd0-4632-a502-e61fb624a8c4.png">
